grpc/server/admin_test.go

```diff
@@ -99,3 +166,123 @@ func TestAdminServer_GetAccessToken(t *testing.T) {
 		})
 	}
 }
+
+func TestAdminServer_ListAccessTokens(t *testing.T) {
+	tests := []struct {
+		name              string
+		token             *commonv1.AccessToken
```

Comment:
Is this token actually needed/used as part of the test?

Reply:
this is used for the `mockQuerierResult` data and for the client result assertion at the end of test